app/default/routes.py

```diff
@@ -23,9 +24,7 @@ def index():
 @assess_bp.errorhandler(404)
 @default_bp.errorhandler(404)
 def not_found(error):
-    error_message = f"Encountered 404: {error}"
-    stack_trace = traceback.format_exc()
-    current_app.logger.info(f"{error_message}\n{stack_trace}")
+    current_app.logger.info(f"Encountered 404 against url {request.path}")
```

Comment:
I like this change.  I don't think we need full stack traces for 404's, that one was my bad.  Should stop us from scrolling a lot through the logs to find errors...